imsv-docs-docusaurus/openapi/endpoints/transactions/models/transaction.yaml

```diff
@@ -83,6 +83,18 @@ properties:
     enum:
       - "credit"
       - "debit"
+  paymentType:
+    type: string
+    description: The type of payment
+    enum:
+      - "purchase"
+      - "refund"
+      - "adjustment"
+    example: "purchase"
+  relatedPaymentId:
+    type: string
+    description: The ID of the related transaction, if applicable
```

Comment:
related**Payment**Id - Yet im using `transaction` in the description. 
Shall we just call it `Payment` from here onwards? I am worried that using these two terms interchangbly causes confusion